rclcpp/test/rclcpp/strategies/test_allocator_memory_strategy.cpp

```diff
@@ -511,9 +511,7 @@ TEST_F(TestAllocatorMemoryStrategy, number_of_entities_with_subscription) {
   RclWaitSetSizes expected_sizes = {};
   expected_sizes.size_of_subscriptions = 1;
   const std::string implementation_identifier = rmw_get_implementation_identifier();
-  if (implementation_identifier == "rmw_connext_cpp" ||
-    implementation_identifier == "rmw_cyclonedds_cpp")
-  {
+  if (implementation_identifier == "rmw_cyclonedds_cpp") {
```

Comment:
IMO, keeping `rmw_connext_cpp` here doesn't make any harm.
Not removing support for `rmw_connext_cpp` immediately will make the transition easier IMO